app/javascript/mastodon/locales/pl.json

```diff
@@ -73,7 +73,7 @@
   "emoji_button.search": "Szukaj...",
   "emoji_button.symbols": "Symbole",
   "emoji_button.travel": "Podróże i miejsca",
-  "empty_column.community": "Lokalna oś czasu jest pusta. Napisz coś publicznie, aby odbić piłeczkę!",
+  "empty_column.community": "Lokalna oś czasu jest pusta. Napisz coś publicznie, aby zagaić!",
```

Comment:
The previous phrase „odbić piłeczkę” roughly refers to responding in Polish, hence doesn’t look appropriate for a situation in which there is no activity yet.